src/pool/srv_pool.c

```diff
@@ -1410,6 +1439,47 @@ pool_prop_read(struct rdb_tx *tx, const struct pool_svc *svc, uint64_t bits,
 		       value.iov_buf_len);
 		idx++;
 	}
+	if (bits & DAOS_PO_QUERY_PROP_OWNER) {
+		daos_iov_set(&value, NULL, 0);
+		rc = rdb_tx_lookup(tx, &svc->ps_root, &ds_pool_prop_owner,
+				   &value);
+		if (rc != 0)
+			return rc;
+		/* IOV len includes strlen + null terminator */
```

Comment:
Looks like the owner and owner_group values written by pool_prop_write do not include the '\0'. Or, did I misread the code?

Reply:
Good catch. Will fix.